docs/VictoriaLogs/Roadmap.md

```diff
@@ -17,8 +17,6 @@ The following functionality is planned in the future versions of VictoriaLogs:
 
 - Support for [data ingestion](https://docs.victoriametrics.com/VictoriaLogs/data-ingestion/) from popular log collectors and formats:
   - Promtail (aka Grafana Loki)
-  - Vector.dev
-  - Fluentbit
```

Comment:
should this be removed?

Reply:
Yes, i've removed it from roadmap, because we already support it

Reply:
ahh, sorry, missed that it is roadmap